cfme/tests/infrastructure/test_vm_retirement_rest.py

```diff
@@ -118,3 +153,46 @@ def _finished():
         return True
 
     wait_for(_finished, num_sec=1500, delay=10, message="REST vm retire future")
+
+
+@pytest.mark.tier(1)
+def test_check_vm_retirement_requester(
+    appliance, request, provider, vm_retirement_report
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/2h
+        tags: retirement
+        setup:
+            1. Add a provider.
+            2. Provision a VM.
+            3. Once the VM has been provisioned, retire the VM.
+            4. Create a report(See attachment in BZ).
+        testSteps:
+            1. Queue the report once the VM has retired
+                and check the retirement_requester column for the VM.
+        expectedResults:
+            1. Requester name must be visible.
+
+    Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1638502
+    """
+    vm_name, report = vm_retirement_report
```

Comment:
It would be better to write it like:
[Please refer this comment](https://github.com/ManageIQ/integration_tests/pull/8675#discussion_r272506251)
```
Bugzilla:
    1638502
```